### PR TITLE
making sparkline svg tag use same dimension as option

### PR DESCRIPTION
**Pull request by davidandrus**

### CODE REVIEW

File: lib/sparkLine.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ var spark = module.exports = Base.extend({
     var data = this.attributes.data,
         opts = this.attributes.opts,
         svg = this._make('svg',{
-            width: '100%',
+            width: ops.chart.width,
```
Comment (lancekruegger) [inline lib/sparkLine.js]: Is this a typo? Shouldn't it read ```opts.chart.width``` ?

**Comment by hasalfred** [comment]
@lancekruegger your correct

**Comment by hasalfred** [comment]
@lancekruegger fixed under this commit https://github.com/alfredkam/yakojs/commit/fe6a2ed06126c4d91d02b585c6c37f15edae1759

should be out in npm @0.1.3

**Comment by lancekruegger** [comment]
Great! :+1:

**Pull request merged**